scipy/stats/tests/test_stats.py

```diff
@@ -2621,31 +2624,44 @@ def __array__(self, dtype=None, copy=None):
         with pytest.raises(TypeError, match=message):
             stats.mode(np.arange(3, dtype=object))
 
+
+@array_api_compatible
 class TestSEM:
 
-    testcase = [1, 2, 3, 4]
+    testcase = [1., 2., 3., 4.]
     scalar_testcase = 4.
 
-    def test_sem(self):
+    def test_sem(self, xp):
         # This is not in R, so used:
         #     sqrt(var(testcase)*3/4)/sqrt(3)
 
         # y = stats.sem(self.shoes[0])
         # assert_approx_equal(y,0.775177399)
+        scalar_testcase = xp.asarray(self.scalar_testcase)
```

Comment:
Technically
```suggestion
        scalar_testcase = xp.asarray(self.scalar_testcase)[()]
```
to leave the test essentially the same for NumPy.